### PR TITLE
Chore: Add some scripts to manage data locally

### PR DESCRIPTION
**Pull request by Merkur39**
```
### 🔧 Tech

* Add some scripts to manage data locally
```

- [x] Update Readme

### CODE REVIEW

File: scripts/save-random-trip.js [hunk 1]
```diff
@@ -58,7 +58,11 @@ const main = async () => {
   let sourceAccount
   if (!args.source_account) {
     const { data: accounts } = await client.query(
-      Q('io.cozy.accounts').where({ account_type: 'tracemob' })
+      Q('io.cozy.accounts').where({
+        account_type: {
+          $or: ['tracemob', 'openpath']
+        }
+      })
```
Comment (Crash--) [inline scripts/save-random-trip.js]: should be a partialIndex ;)

File: scripts/drop-accounts.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+  if (userConfirmation !== 'yes') {
+    console.log('Aborted')
+    readline.close()
+    return
+  }
+
+  const client = await createClientInteractive({
+    scope: [ACCOUNTS_DOCTYPE, CCO2_SETTINGS_DOCTYPE],
+    uri: 'http://cozy.localhost:8080',
```
Comment (Crash--) [inline scripts/drop-accounts.js:40]: ça devrait pas venir des args l'uri ?

File: scripts/save-account.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  })
+  parser.add_argument('-u', '--url', {
+    default: 'http://cozy.localhost:8080',
+    help: 'Use custom url. Default: http://cozy.localhost:8080'
+  })
+  const args = parser.parse_args()
+
+  const client = await createClientInteractive({
+    scope: [ACCOUNTS_DOCTYPE],
+    uri: args.url,
```
Comment (Crash--) [inline scripts/save-account.js:34]: ah bah ici c'est bien géré

File: scripts/drop-accounts.js (hunk 2 above)
Comment (Merkur39) [reply scripts/drop-accounts.js]: My bad, tout à fait!

File: scripts/save-random-trip.js (hunk 1 above)
Comment (Merkur39) [reply scripts/save-random-trip.js]: Nice catch, I also fixed the drop-account and drop-accounts scripts

**Review by Crash-- — APPROVED** [review]
Nice!

File: scripts/save-account.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    }
+  })
+
+  const accountType = ACCOUNT_TYPES[crypto.randomInt(0, ACCOUNT_TYPES.length)]
+  const account = {
+    _type: ACCOUNTS_DOCTYPE,
+    _id: crypto.randomBytes(16).toString('hex'),
+    account_type: args.account_type || accountType,
+    auth: {
+      credentials_encrypted: crypto.randomBytes(16).toString('hex'),
```
Comment (paultranvan) [inline scripts/save-account.js:47]: The new openpath accounts do not use `auth.credentials_encrypted` but a `token` attribute. It still works, but this is meant to progressively disappear

Comment (paultranvan) [reply scripts/save-account.js:47]: We could also eventually add an option to manually specify the token with the CLI

**Review by paultranvan — COMMENTED** [review]
That's really great, thanks! :clap:

File: scripts/save-random-trip.js (hunk 1 above)
Comment (paultranvan) [reply scripts/save-random-trip.js]: True, but not a big deal for tooling, and even less for small databases such as accounts, IMO ;)

**Pull request merged**